tests/test_symbols.py

```diff
@@ -1788,6 +1787,43 @@ def test_state_size(self):
         self.assertEqual(model.set(10, max_size=5).state_size(), 5 * 8)
 
 
+class TestSize(utils.SymbolTests):
+    def generate_symbols(self):
+        model = Model()
+
+        a = dwave.optimization.symbols.Size(model.constant(5))
+        b = dwave.optimization.symbols.Size(model.constant([0, 1, 2]))
+        c = model.set(5).size()
+
+        with model.lock():
+            yield a
+            yield b
+            yield c
+
+    def test_dynamic(self):
```

Comment:
I'm trying to get the length for disjoint_lists:
```
model = Model()
model.states.resize(1)

x, ys = model.disjoint_lists(5, 3)
length0 = ys[0].size()
length1 = ys[1].size()
length2 = ys[2].size()

x.set_state(0, [[0, 1, 2, 3, 4], [], []])
with model.lock():
    print(length0.state(0))
    print(length1.state(0))
    print(length2.state(0))
```
Is there a better way for doing this? For example, can we make an array of all lengths?

Reply:
Currently no way to join them into another array unfortunately, we'd need #46 for that I think